input/pagecontent/module_family.md

```diff
@@ -1 +1,21 @@
-### Family
+# Family
+
+## Overview
+This module describes family groups and family relationships, often used in genomic analyses to aid in identification of causal variants. We include several articulations of these: Family Study is an entity that describes a collection of participants in the context of a research study. Family Relationships describe the relationship of two Participants, and should be sufficient for generating a pedigree if the source information is present.
+
+This does not include specific methods for managing reports of “family history”. Those elements can either be encoded as specific “Participants” for which data are recorded as assertions and their family relationship to the Participant are recorded OR encoded as assertions from the Participant (eg, “family history of hypertension” or “Maternal history of hypertension”).
+
```

Comment:
I think we should probably note awareness of the GA4GH efforts on family health history and KIN ontology: https://github.com/ga4gh/pedigree-fhir-ig https://github.com/ga4gh/pedigree_family_history_terminology and that part of the work to do is alignment between these efforts which are also anchored on being useful in genetic analysis.

Reply:
@allisonheath I added some language here on line 8

Reply:
just documenting that i updated this language based on Allison's feedback